test/integration/query-interface/changeColumn.test.js

```diff
@@ -223,7 +227,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(describedTable.level_id.allowNull).to.not.equal(firstTable.level_id.allowNull);
           expect(describedTable.level_id.allowNull).to.be.equal(true);
         });
-        if (dialect !== 'db2') {
+        if (dialect !== 'db2' && dialect !== 'oracle') {
```

Comment:
Can you document whether oracle doesn't support column comments at all, or changing them?

Reply:
The Oracle dialect doesn't support comment